### DISCUSSION TITLE
Hide tenants in default admin?

### DISCUSSION THREAD

**Discussion by fladi** [lorinkoz/django-pgschemas#281] [Q&A] [answered]
Hi, I have my tenant and domain models in my public schema and have enabled Django admin app in the default tenant. Now if I log in to a tenant admin, I can also see the tenant model admin there. What is the recommended way to disable it so tenant admins cannot see or modify the other tenants.

**Comment by fladi** [comment]
Calling `admin.site.unregister(Tenant)` removes the Tenant model from the admin site when I all this in the tenant admin.py. Is this sufficient?

**Reply by lorinkoz** [reply, marked as answer]
You can have different admin sites. I'd recommend you have one site for tenant management, another site for the dynamic tenants. There you can register models at will.

Related docs: https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#adminsite-objects
